tests/ztest_test.go

```diff
@@ -1,3 +1,5 @@
+// +build system
+
```

Comment:
I believe the goal of this is to prevent `go test ./...` from running the new script ztests.

It has the side effect of preventing `go test ./...` from running the "in-process" ZQL ztests. I don't think we want that.

One way to achieve the goal without the side effect is to replace the ZTEST_ZQ environment variable with, say, ZTEST_PATH and condition running the script ztests on its presence.

Reply:
this is perfect, thanks.